modules/op-deleting-the-pipelines-component-and-custom-resources.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// */openshift_pipelines/uninstalling-pipelines.adoc
+
+[id='op-deleting-the-pipelines-component-and-custom-resources_{context}']
+= Deleting the {pipelines-title} components and custom resources
+
+Delete the custom resources created by default during installation of the {pipelines-title} Operator.
+
+[discrete]
+== Procedure
+. In the *Administrator* perspective of the Web Console, navigate to *Operators->Installed Operators*.
+
+. In the *Project* drop-down select the *OpenShift-Operators* project.
```

Comment:
I think that this is not necessary because the operator is installed in all namespaces.

Reply:
removed this section, and added new instructions